src/curryN.js

```diff
@@ -38,9 +38,7 @@ var _curryN = require('./internal/_curryN');
  * @see R.curry
  * @example
  *
- *      var addFourNumbers = function() {
- *        return R.sum([].slice.call(arguments, 0, 4));
- *      };
+ *      var addFourNumbers = (...args) => R.sum(args.slice(0, 4));
```

Comment:
Oh, I see. We're manufacturing an example for which `R.curry` is unsuitable. I suggest we replace `addFourNumbers` with a function which returns the sum of *all* its arguments:

```javascript
var sumArgs = (...args) => R.sum(args);

var curriedAddFourNumbers = R.curryN(4, sumArgs);
```